src/components/comment/Comment.tsx

```diff
@@ -52,30 +54,65 @@ const Comment: React.FC<CommentProps> = ({ responseText, details, addResponseVot
     setIsDeleted(true)
   }
 
+  const editComment = () => {
+    setIsEditing(true)
+  }
+
+  const packageNewCommentText = () => {
+    return {
+      body: commentBody,
+    }
+  }
+
+  const handleCommentEditSubmit = () => {
+    const newCommentText = packageNewCommentText()
+    updateCommentText(responseText.id, newCommentText)
+    setIsEditing(false)
+  }
+
   return (
     <div className='Comment--container'>
-    {!isEditing && (
```

Comment:
I see you added editing for comment and response! Awesome job! .. We now need to add the user context and tie this to the user. We will have to add an issue for this and get this done next. :)